src/pages/journal/journal-components/journal-time/__tests__/journal-time.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { JournalTimeComponent } from '../journal-time';
+import { IonicModule } from 'ionic-angular';
+import { DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+
+
+describe('JournalTimeComponent', () => {
+  let component: JournalTimeComponent;
+  let fixture: ComponentFixture<JournalTimeComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [JournalTimeComponent],
+      imports: [IonicModule]
+    })
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(JournalTimeComponent);
+        component = fixture.componentInstance;
+        component.time =  '2018-12-10T10:04:00+00:00';
+        component.testComplete = true;
+      });
+  }));
+
+  describe('Class', () => {
+    it('should create', () => {
+      expect(component).toBeDefined();
+    });
+
+  describe('DOM', () => {
```

Comment:
Could do with a test for the `ngClass` here.

Reply:
Added test for ngClass.